client/ngapp/views/about.html

```diff
@@ -23,10 +23,10 @@
     <p>
     To contact the OONI team send an email to <a href="mailto:contact [AT] ooni.io">contact [AT] ooni.io</a>.
```

Comment:
I've also marked `openobservatory.org` as primary UID for `6B2943F00CB177B7` (`ooni.io` was primary one). But I'm unsure if it's properly uploaded to keyservers.